Addition to cover the dictionary feature (#857)
akeneo/pim-helpcenter: content/md/first-steps/manage-data-quality.md

![Image data quality attribute groups reminder](../img/data-quality-attribute-groups-reminder.png)
:::

> # Manage your dictionary

Reviewer: Aren't they multiple dictionaries? One per locale? If yes, then:
```suggestion
# Manage your dictionaries
```

> Either because your product names are unique or due to industry acronyms that are not part of regular dictionaries, you can add words in your personal dictionary to fit your special needs and avoid miscalculation of your Quality Score.

Reviewer: Can we add a link on the "Quality score" words to link to the article that talks about it?

Several ways to add words to the `Dictionary`:
- Through the "ignore" feature in the spell-check pop-in displayed when a possible typo error is detected
- In the `Locales` settings screen

![Image data quality spelling](../img/data-quality-spelling.png)
:::

:::info

> To save you time, in addition to the manual ways, Akeneo PIM searches all words used in your product data (in text, text area attributes and attribute and option labels. If a word that counts a minimum of 3 letters is found at least 10 times, then this word will be automatically added to your personal dictionary.)

Reviewer: ```suggestion
To save you time, in addition to the manual ways, Akeneo PIM searches all words used in your product data (in text, text area attributes and attribute and option labels). If a word that counts a minimum of 3 letters is found at least 10 times, then this word will be automatically added to your personal dictionary.
```

:::

## Locales dictionary
For users with rights to edit locales, it is possible to add, edit or delete words in the dictionary.

> Go to `Settings/Locales/Dictionary`

Reviewer: Shouldn't they need to select a locale first?

![Image data quality dictionary](../img/data-quality-dictionary.png)
:::

> Several ways to add words from the `Dictionary` screens

Reviewer: ```suggestion
Several ways to add words from the `Dictionary` screens:
```

- For several locales, by clicking on a locale name and using the mass action to add the same word or words for the selected locales

> - For a single  locale, by using the "add words" box

Reviewer 2: ```suggestion
- For a single locale, by using the "add words" box
```

![Image data quality add words to dictionary](../img/data-quality-add-words-dictionary.png)

:::info
To add several words at a time, separate them with comas "," or space.

> You can add up to 100 words at a time.

Reviewer 2: This limit have been removed from the "TagInput" component in the DSM, so we can ignore this rule.

```suggestion
```

> Words are not case-sensitive: you can either type them all in lower or uppercases.

Reviewer 2: ```suggestion
Words are case-insensitive: you can either type them all in lower or uppercases.
```
